### DISCUSSION TITLE
Precision - Digits

### DISCUSSION THREAD

**Discussion by karthikkommalapati** [dirkschumacher/ompr#360] [Q&A]
Hello,

I am having an issue when using the **ompr** model,  it seems the model restricts the objective value by **7** digits? The challenge I have is I am using output from one model as the boundary to another model the result from model one is as expected, but once the objective value is given as a constraint boundary to a second model the results vary. It seems like the objective matrix is restricted to **7** digits by default. I am looking for a precision value of up to 4 decimal places, for example, 100000.1234 the objective matrix is only considering 7 digits (100000.1) altogether. Grateful if you please let me know if there is a workaround for this?

Kind Regards,
Karthik

**Comment by dirkschumacher** [comment]
The precision used is probably due to the solver being used. Can you post a [minimal, reproducible example](https://www.tidyverse.org/help/)? I will move this to Q&A in the discussions.
